index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>    
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css" type="text/css"/>
+    <title>Mobile Portfolio</title>
+</head>
+<body>
+    <div class="container">
+    <div class="wrapper">
+        <header><h1>Alfred Mobile Portfolio</h1></header>
+        
+          <button class="btn1 btn" type="button"> HTML/CSS</button>  
+          <button class="btn2 btn" type="button">Ruby on Rails</button>  
+          <button class="btn3 btn" type="button">JavaScript</button> 
+                 
+        
+        <div class="sec"><img src="image/sc.png" alt="logo"/></div>        
+        <div class="headline"> 
+            <button class="img1 im" type="button"><img src="image/sc.png" alt="logo"/></button>  
+            <button class="img2 im" type="button"><img src="image/sc.png" alt="logo"/></button>  
+            <button class="ing3 im" type="button"><img src="image/sc.png" alt="logo"/></button>
+            <button class="img4 im" type="button"><img src="image/sc.png" alt="logo"/></button> 
+        </div> 
```

Comment:
- [ ] The headline should be in a section tag.
- [ ] The section has the proper background image (decorations are always background images, not `img` tags).
- [ ] The headline should have The subtitle (if existing) uses a header tag `hx (examples: <h1>,<h2>,<h3>, etc...)`.
- [ ] The headline should have the list of icons that uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).
- [ ] The headline should aligned with flexbox.
- [ ] The headline should has the margins (top, bottom, left, right) defined in the design.